.teamcity/_self/projects/WPComTests.kt

```diff
@@ -404,15 +404,13 @@ private object VisualRegressionTests : BuildType({
 			notifierSettings = slackNotifier {
 				connection = "PROJECT_EXT_11"
 				sendTo = "#visual-regression-automated-tests"
-				messageFormat = verboseMessageFormat {
-	            	addBranch = false
-	            }
+				messageFormat = simpleMessageFormat()
 			}
-			buildFailedToStart = true
+			branchFilter = """
```

Comment:
The indentation is weird. Also I think you can just do `branchFilter = "trunk"`

Reply:
Yeah, you got here too quickly! I just noticed the spacing. Updated the branchfilter too. thanks!